src/Take.Blip.Builder/Hosting/IConfiguration.cs

```diff
@@ -29,5 +31,7 @@ public interface IConfiguration
         long ExecuteScriptLimitMemoryWarning { get; }
 
         TimeSpan ExecuteScriptTimeout { get; }
+
+        string IpsDeniedOnHttpAction { get; }
```

Comment:
Suggestion: in order to keep the naming consistency already started for additional settings for "ExecuteScript" action - where the name of the action comes first, this new property should be named ProcessHttp<something>.

And since not only Ips are allowed, I suggest ProcessHttpDeniedHosts instead.